pkg/templates/templates/credentials/create/credential-set.yaml

```diff
@@ -0,0 +1,22 @@
+schemaType: CredentialSet
+schemaVersion: 1.0.0
+name: NAME
+namespace: NAMESPACE
+labels:
+  LABEL: LABEL_VALUE
+credentials:
+  - name: credential-path
+    source:
+      path: /path/to/credential-path-file.txt
+  - name: credential-command
+    source:
+      command: echo 'credential command'
+  - name: credential-env
+    source:
+      env: CREDENTIAL_ENV
+  - name: credential-value
+    source:
+      value: credentialvalue
+  - name: credential-secret
+    source:
+      value: credential-secret
```

Comment:
I was hoping that we could have inline comments that would help someone know what to do with this file.
```suggestion
# See the Credential Set file format documentation at https://release-v1.porter.sh/reference/file-formats/#credential-set
schemaType: CredentialSet
schemaVersion: 1.0.0
# Name of the credential set.
name: NAME
# The namespace in which the credential set is defined.
# Omit the namespace to define the credential set in the current current namespace.
namespace: NAMESPACE
# Optionally define labels which can be used for filtering. Labels must be string values.
labels:
  LABEL: LABEL_VALUE
# Credentials specify how Porter can resolve credentials when executing a bundle.
# Allowed sources are: value, path, env, secret, and command.
credentials:
  - name: credential-path
    source:
      # Resolve the credential value from the contents of the specified file.
      path: /path/to/credential-path-file.txt
  - name: credential-command
    source:
      # Resolve the credential value from the output of the specified command.
      command: echo 'credential command'
  - name: credential-env
    source:
      # Resolve the credential value from the value of the specified environment variable.
      env: CREDENTIAL_ENV
  - name: credential-value
    source:
      # Use the specified value for the credential. Do not use "value" for sensitive data.
      value: credentialvalue
  - name: credential-secret
    source:
      # Resolve the credential value from the secret store specified in your Porter configuration file.
      secret: credential-secret
```

Reply:
On line 34, it should be "secret", not "value".

Reply:
Ah, sorry for the "secret" typo. And thank you for adding the recommendations for the inline comments as well, Carolyn!

Reply:
The documentation site that we put in the inline comment for the people to refer to is the `release-v1` one (https://release-v1.porter.sh/reference/file-formats/#credential-set), should we add a TODO here to change it back to just porter.sh after the v1 got merged in the future? Since I assume that we will deprecate the `release-v1.porter.sh` site after it got merged (?)

Reply:
Instead of adding TODO's everywhere in the code (which is easy to forget!) I've added an issue and added it to the v1 milestone https://github.com/getporter/porter/issues/1949